qiskit_experiments/database_service/fit_value_type.py

```diff
@@ -0,0 +1,90 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Extended value."""
+
+from typing import Optional
+
+import numpy as np
+
+
+# pylint: disable=invalid-name
+class fitval:
+    """Extended float type to support value error and unit.
+
+    This value is serializable with the Qiskit Experiment json serializer.
+    """
+
+    def __init__(
+        self,
+        value: float,
+        stdev: Optional[float] = None,
+        unit: Optional[str] = "a.u.",
+    ):
+        """Create new fit value instance.
+
+        Args:
+            value: Value.
+            stdev: Optional. Standard deviation of value.
```

Comment:
Might be better to call this standard error or just error
```suggestion
            stderr: Optional. Standard error of value.
```